omnibus/package-scripts/iot-agent/postinst

```diff
@@ -14,79 +16,91 @@ DISTRIBUTION=$(lsb_release -d 2>/dev/null | grep -Eo $KNOWN_DISTRIBUTION || grep
+            SYSTEMCTL_SKIP_SYSV=true systemctl enable $SERVICE_NAME || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with systemctl"
+        elif command -v initctl >/dev/null 2>&1; then
+            # Nothing to do, this is defined directly in the upstart job file
+            :
+        elif command -v update-rc.d >/dev/null 2>&1; then
+            update-rc.d $SERVICE_NAME defaults || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with update-rc.d"
+        else
+            echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd, upstart and sysvinit."
+        fi
 
-	# Enable and restart the agent service here on Debian platforms
-	# Only supports systemd and upstart
-	echo "Enabling service $SERVICE_NAME"
-	if command -v systemctl >/dev/null 2>&1; then
-		systemctl enable $SERVICE_NAME || echo "[ WARNING ]\tCannot enable $SERVICE_NAME with systemctl"
-	elif command -v initctl >/dev/null 2>&1; then
-		# Nothing to do, this is defined directly in the upstart job file
-		:
-	else
-		echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd and upstart."
-	fi
-
-	# TODO: Use a configcheck command on the agent to determine if it's safe to restart,
-	# and avoid restarting when a check conf is invalid
-	if [ -f "$CONFIG_DIR/datadog.yaml" ]; then
-		echo "(Re)starting $SERVICE_NAME now..."
-		if command -v systemctl >/dev/null 2>&1; then
-			systemctl restart $SERVICE_NAME || true
-		elif command -v initctl >/dev/null 2>&1; then
-			initctl start $SERVICE_NAME || initctl restart $SERVICE_NAME || true
-		else
-			echo "[ WARNING ]\tCannot detect a supported init system. The datadog-agent package only provides service files for systemd and upstart."
-		fi
-	else
-		# No datadog.yaml file is present. This is probably a clean install made with the
-		# step-by-step instructions/an automation tool, and the config file will be added next.
-		echo "No datadog.yaml file detected, not starting the agent"
-	fi
+        # TODO: Use a configcheck command on the agent to determine if it's safe to restart,
+        # and avoid restarting when a check conf is invalid
+        if [ -f "$CONFIG_DIR/datadog.yaml" ]; then
+            echo "(Re)starting $SERVICE_NAME now..."
+            if command -v systemctl >/dev/null 2>&1; then
+                systemctl restart $SERVICE_NAME || true
+            elif command -v initctl >/dev/null 2>&1; then
+                initctl start $SERVICE_NAME || initctl restart $SERVICE_NAME || true
+            elif command -v service >/dev/null 2>&1; then
+                service $SERVICE_NAME restart || true
+            else
+                echo "[ WARNING ]\tCannot detect a supported init system. The datadog-iot-agent package only provides service files for systemd, upstart and sysvinit."
```

Comment:
Same here: do we support `sysvinit` for the `iot-agent` package?

Reply:
We don't, it's probably an oversight from copy-pasting.